### DISCUSSION TITLE
Ip6tables

### DISCUSSION THREAD

**Discussion by cryptcody** [evilsocket/opensnitch#758] [Development]
Ip6tables. table error "mangle"
-A opensnitch-filter-OUTPUT -p icmp -j ACCEPT
To correct 
-A opensnitch-filter-OUTPUT -p icmpv6 -j ACCEPT

Where and how to make changes to generate correctly?

**Comment by gustavo-iniguez-goya** [comment]
Hi @cryptcody ,

See here: https://github.com/evilsocket/opensnitch/wiki/System-rules-legacy#allowing-vpn-traffic-and-other-things

**Reply by cryptcody** [reply]
Thank you friend!
I initially tried to do this, but for some reason opensnitchd did not start for me.  maybe I made a mistake .. I'll try to figure it out.

**Reply by cryptcody** [reply]
I apologize, my mistake was not to put a comma.
Thanks for the link.  She helped me a lot.

**Reply by gustavo-iniguez-goya** [reply]
see if there's any errors in the log /var/log/opensnitchd.log, and verify that the file /etc/opensnitchd/system-fw.json is correct (jsonlint.com for example)
